independent-publisher-connectors/SOS Inventory/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2218 @@
+				"description": "List Pick Tickets",
+				"operationId": "PickTicket",
+				"x-ms-visibility": "important",
+				"parameters": [{
+						"in": "query",
+						"name": "start",
+						"type": "number",
+						"x-ms-summary": "start",
+						"description": "A cursor used in pagination",
+						"required": false,
+						"x-ms-visibility": "advanced"
+					},
+					{
+						"in": "query",
+						"name": "maxresults",
+						"type": "number",
+						"x-ms-summary": "Limit",
+						"description": "Maximum number of results to return",
+						"required": false,
+						"x-ms-visibility": "advanced",
+						"default": "200"
+					},
+					{
+						"in": "query",
+						"name": "summary",
+						"type": "string",
+						"x-ms-summary": "Summary",
+						"description": "If present, returns only summary values.",
+						"required": false,
+						"x-ms-visibility": "advanced"
+					},
+					{
+						"name": "query",
+						"in": "query",
+						"x-ms-summary": "query",
+						"description": "Filter results by matching query.",
+						"type": "string",
+						"x-ms-visibility": "advanced"
+					},
+					{
+						"in": "query",
+						"name": "archived",
+						"type": "boolean",
+						"x-ms-summary": "archived",
+						"description": "Filters by archived or not.",
+						"required": false,
+						"x-ms-visibility": "advanced"
+					},
+					{
+						"name": "from",
+						"in": "query",
+						"x-ms-summary": "From Date",
+						"description": "Starting transaction date.",
+						"type": "string",
+						"format": "date",
+						"x-ms-visibility": "important"
+					},
+					{
+						"name": "to",
+						"in": "query",
+						"x-ms-summary": "To Date",
+						"description": "Ending transaction date.",
+						"type": "string",
+						"format": "date",
+						"x-ms-visibility": "advanced"
+					},
+					{
+						"name": "location",
+						"in": "query",
+						"x-ms-summary": "Location",
+						"description": "Filters according to location.",
+						"type": "string",
+						"x-ms-visibility": "advanced"
+					},
+					{
+						"name": "status",
+						"in": "query",
+						"x-ms-summary": "Location",
+						"description": "Filters according to open/closed.",
+						"type": "string",
+						"enum": [
+							"open",
+							"closed"
+						],
+						"x-ms-visibility": "advanced"
+					}
+				]
+			}
+		}
+	},
+	"x-ms-connector-metadata": [{
+			"propertyName": "Website",
+			"propertyValue": "https://thaiwala.com"
+		},
+		{
+			"propertyName": "Privacy policy",
+			"propertyValue": "https://thaiwala.com"
+		},
+		{
+			"propertyName": "Categories",
+			"propertyValue": "Business Management"
+		}
+	],
+	"definitions": {
+		"salesorder": {
+			"type": "object",
+			"properties": {
+				"id": {
+					"type": "number",
+					"description": "Sales Order ID",
+					"example": "80"
+				},
+				"starred": {
+					"type": "number",
+					"example": "0"
+				},
+				"syncToken": {
+					"type": "number",
+					"description": "Sales Order syncToken",
```

Comment:
@Harold-Anderson this value has CaMel case word too. If you want to keep it as it is, it's fine. If not, we need to fix the casing. More information about non-translatable words can be found in [documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).